test/native-tests/test-python/helloWorld.py

```diff
@@ -13,7 +13,7 @@
     platform = "so"
 
 build_type = sys.argv[1]
-if sys.argv[2] != None:
+if sys.argv.__len__() > 2 and sys.argv[2] != None:
```

Comment:
I believe so.  I was getting failures when running the tests that used this about teh index being out of bounds.  Adding this checked made that error go away.